src/directives/formly-field.test.js

```diff
@@ -1014,6 +1014,32 @@ describe('formly-field', function() {
         expect(ctrl.$formatters).to.have.length(2); // ngModel adds one
         expect(ctrl.$viewValue).to.equal('hello! boo!');
       });
+
+      it.skip(`should format a model value right from the start`, () => {
+        scope.model = {myKey: 'hello'};
+        scope.fields = [getNewField({
+          key: 'myKey',
+          formatters: ['"!" + $viewValue + "!"']
+        })];
+        compileAndDigest();
+
+        const ctrl = getNgModelCtrl();
+
+        //workaround: this formats the model value right off the bat,
+        //without the workaround, the initial model value stays unformatted
+
+/*
```

Comment:
The comment block is the workaround, I'll remove it and add it to the issue comments in stead.